src/module_PI/Raspberry_PI/main/PiApp.java

```diff
@@ -1,37 +1,46 @@
 package module_PI.Raspberry_PI.main;
 
+import java.io.*;
+import java.util.*;
 
 public class PiApp {
 
 	/**
 	 * @param args
 	 */
 	public static void main(String[] args) {
+		parseArgs(args);
 		SerialInputListener inputListener = new SerialInputListener();
 		Thread listener = new Thread(inputListener, "Listener");
 		listener.start();		
 		SerialOutput output = new SerialOutput();
+    	EmailNotifier notifier = new EmailNotifier("lastrellik@gmail.com", "Today's message");
```

Comment:
Oh, this was left by accident. I think I should put more effort into reading over my code again before I commit and push it. This was after 1 AM last night and I was pretty tired and only half paying attention because my girlfriend was watching a tv show.